roboflow/models/inference.py

```diff
@@ -2,8 +2,8 @@
 import json
 import os
 import time
-import urllib
```

Comment:
I think pyright is enforcing this change, but it is mostly cosmetic, with minor performance gains. I prefer to keep it scoped on `urllib` for readability.